### DISCUSSION TITLE
How do I set a target to be pinged by only a slave, not master.

### DISCUSSION THREAD

**Discussion by shamefulCake1** [oetiker/SmokePing#395] [Q&A]
I want to ping "default gateway", basically, from each of the machines.

I actually know the ip of the machine (and if I do not know, I can find out via dns), but it does not make much sense to ping 192.168.2.1 from the master when it is miles away from the actual 192.168.2.1.

When, on a target, I write `slaves = slave1` this does not stop master from pinging the same host.
I can, I guess, make a fake graph with `host = 192.168.2.1~slave1`  but this would just hide the useless pinger, not disable it.

**Comment by knofte** [comment]
@shamefulCake1 Have you tried this flag:
nomaster = true  # Ensure the master does not ping this host
    $return{nomasterpoll} = 'no'; # slaves poll always

Not sure if the config option is "nomasterpoll = no" or not here though.

I have not tried it myself, but give it a whirl please and see.
